### DISCUSSION TITLE
Ico quality on Stream Deck +

### DISCUSSION THREAD

**Discussion by djdavi** [nguyenquyhy/Flight-Tracker-StreamDeck#430] [Q&A] [answered]
Hi, I show with an example my question:

![20241130_141147](https://github.com/user-attachments/assets/df0f03fa-2761-4789-bc66-81b0dd928e8c)
![20241130_141157](https://github.com/user-attachments/assets/111f7669-8dba-40df-b011-3e346a00a5e0)

The button above is normal Stream Deck Hotkey, under we have the Generic Toggle Button Custom ON/OFF image.
I see a difference, Stream Deck default button is more detailed.
I used the same ICO png file (288x288).
I try to use 144x144 too, but with same results.

There is a way to have better resolution buttons?

**Comment by nguyenquyhy** [comment, marked as answer]
I think all images was scaled down to 72*72 as it was the resolution of the original stream deck. The plugin has to perform that scaling before sending the image to the button as some users put in huge images leading to some performance issue.

I'll look into if I can detect Stream Deck+ and use double resolution instead. I remember doing that before but maybe I have never finished it.

**Reply by djdavi** [reply]
Yes yes I think too.
Ok thank you very much.
It's not a big problem anyway 😅

**Reply by nguyenquyhy** [reply]
Tracked in https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/issues/433

**Reply by nguyenquyhy** [reply]
Version 0.53 is now released with the requested change. Please help to try and let me know if it works for you. https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/releases/tag/v0.53

**Reply by djdavi** [reply]
Thank you so much for the update, tonight when I come back home I will try it and and I'll let you know 👍🏽

**Reply by djdavi** [reply]
Perfect!
The icons automatically appear in 'HD', I didn't have to do anything because I had already uploaded them at the native resolution.
Thank you!
